internal/explorer.point/src/hooks/useLinkPointToSol.js

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+import Swal from 'sweetalert2';
+
+const USER_REJECTION_CODE = 4001;
+
+const successMessage = `
```

Comment:
**Minor & For the future:** let's figure out is there a better way to customise the success dialog.